src/js/components/Pagination/__tests__/Pagination-test.js

```diff
@@ -0,0 +1,38 @@
+// import React from 'react';
+// import 'jest-styled-components';
+// import 'jest-axe/extend-expect';
+// import 'regenerator-runtime/runtime';
+
+import { cleanup } from '@testing-library/react';
+// import { axe } from 'jest-axe';
+// import { Grommet } from '../../Grommet';
+// import { Pagination } from '..';
+
+describe('Pagination', () => {
+  afterEach(cleanup);
+
+  test('should have no accessibility violations', () => {});
+
+  test('should calculate correct number of pages', () => {});
```

Comment:
This seems to be written as implementation oriented rather than results oriented. How can we rephrase but get the desired intent?

```suggestion
  test('should display the correct last page based on items length and step', () => {});
```